packages/bpk-component-page-indicator/src/BpkPageIndicator.module.scss

```diff
@@ -0,0 +1,91 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index.scss';
+
+$bpk-spacing-v2: true;
+$dot-size: bpk-spacing-md();
+$dot-margin: bpk-spacing-sm();
+$displayed-total: 5;
+/* stylelint-disable-next-line scss/operator-no-newline-after */
+$container-width: $dot-size * $displayed-total + $dot-margin * $displayed-total *
+  2;
+
+.bpk-page-indicator {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+
+  &__showNav {
+    justify-content: space-between;
+  }
+
+  &__container {
+    max-width: $container-width;
+    min-height: $dot-size;
+    overflow: hidden;
+  }
+
+  &__indicators-container {
+    --direction: -1;
+
+    /* stylelint-disable scss/operator-no-newline-after */
+    transform: translateX(
+      calc(
+        var(--direction) * var(--scroll-index, 0) * #{$dot-size + $dot-margin *
+          2}
+      )
+    );
+    /* stylelint-enable scss/operator-no-newline-after */
+    transition: transform $bpk-duration-sm ease-in-out;
+    white-space: nowrap;
+
+    @include bpk-rtl {
+      --direction: 1;
+    }
+  }
+
+  &__indicator {
+    display: inline-block;
+    width: $dot-size;
+    height: $dot-size;
+    padding: 0;
+    border: none;
+    border-radius: 50%;
+    margin-inline: $dot-margin;
```

Comment:
I'm not against using it here, but last time we checked CSS logical properties we decided to wait until Safari 14< usage drops: https://github.skyscannertools.net/tech-initiatives/production-standards/pull/48
I think it's fine using it here since it keeps the gallery fully functional, just not as pretty for Safari<14 users

Reply:
Thank you! I think your concerns and suggestions are very good, I did some cursory searches and about <0.8% of users are using safari <14, so I think we should be OK with this solution for now and we'll optimize it in the future when we have a better solution.